packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.tsx

```diff
@@ -115,6 +115,10 @@ export class SearchBox extends BaseComponent<ISearchBoxProps, ISearchBoxState> {
     });
 
     this._events.on(this._rootElement, 'blur', this._onBlur, true);
+
+    if (this.props.onFocus) {
+      this.props.onFocus(ev as React.FocusEvent<HTMLInputElement>);
```

Comment:
_onFocusCapture is being put on the div, not the input element, so the type should be unchanged.